api.md

```diff
@@ -65,6 +65,7 @@ Type: {handleErrorResponse: function (response: [Response](https://developer.moz
 -   `afterSuccess` **function (request: [ApiDataRequest](#apidatarequest), dispatch: [Function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function), getState: function (): [Object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)): void?** 
 -   `timeout` **[number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number)?** 
```

Comment:
Is dit handig? Dit bestand word namelijk automatisch gegenereerd.
Misschien beter om dit in de JSDoc comments te zetten, zoals bv hier al gebeurd: https://github.com/oberonamsterdam/react-api-data/blob/e1dcf9e96706b31910b35c16711be5e8547d32b6/src/withApiData.tsx#L35